http/server.go

```diff
@@ -27,22 +27,22 @@ type Server struct {
 
 	srv     *http.Server
 	signals map[os.Signal]struct{}
-	logger  *zap.Logger
+	log     *zap.Logger
 	wg      sync.WaitGroup
 }
 
 // NewServer returns a new server struct that can be used.
-func NewServer(handler http.Handler, logger *zap.Logger) *Server {
-	if logger == nil {
-		logger = zap.NewNop()
+func NewServer(handler http.Handler, log *zap.Logger) *Server {
+	if log == nil {
```

Comment:
If the RFC says that a non-nil logger should be provided then I think this conditional should be removed.

Reply:
Also the parameter order is incorrect. Fixed.